osc/git_scm/store.py

```diff
@@ -68,11 +68,16 @@ def apiurl(self):
 
     @property
     def project(self):
+        if self._project is None:
+            with open(os.path.join(os.path.join(self.abspath, '.osc/_project'))) as f:
```

Comment:
Extra `os.path.join()`.

Is the `.osc` directory always present?
Couldn't we use it for detecting project and package names?

Reply:
at least always when used "osc co". However, using git cli now to find toplevel directory as dirk recommended.